### PR TITLE
forth: Expect a list rather than a string or Text

### PR DESCRIPTION
**Pull request by petertseng**
Like in https://github.com/exercism/x-common/pull/394, I request that we
question our assumptions. Why have we requested a string? Well, it was
like that in https://github.com/exercism/exercism.io/pull/1188, the
first appearance of this exercise. But we can just use a list, right?

### CODE REVIEW

File: exercises/forth/src/Forth.hs [hunk 1]
```diff
@@ -23,5 +23,5 @@ empty = undefined
 evalText :: Text -> ForthState -> Either ForthError ForthState
 evalText = undefined
 
-formatStack :: ForthState -> Text
-formatStack = undefined
+toList :: ForthState -> Text
```
Comment (rbasso) [inline exercises/forth/src/Forth.hs]: I think that you forgot to change this line to `[Int]`.

**Comment by petertseng** [comment]
> provide a clean `package.yaml`, without a dependency on `containers`.

Sure! I will take the opportunity to do so.

> I think that you forgot to change this line to `[Int]`.

Oh dear, maybe I need to speed up the process of doing the "should-compile-but-fail-tests" and we should throw every stub solution into that category. Would prevent this sort of mistake.

**Comment by petertseng** [comment]
okay, example is moved - note that I didn't know what to put after `success-` so I put... `success-standard`. Maybe just `success-example`? Or maybe simply `success` will do?

Also, maybe I should take a moment to do this for all package.yaml that have extra packages...

File: exercises/forth/examples/success-standard/package.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+dependencies:
+  - base
+  - text
+
+library:
+  exposed-modules: Forth
+  source-dirs: src
+  dependencies:
+    - containers
+
```
Comment (rbasso) [inline exercises/forth/examples/success-standard/package.yaml]: Great...you even remembered to remove the comments. :)

**Comment by petertseng** [comment]
OK, test passed which is what I was waiting for. If we have no better suggestion than `success-standard` then I'll merge this one.

**Comment by rbasso** [comment]
`success-standard` is good, don't worry! :+1:

**Pull request merged**